web/client/epics/layers.js

```diff
@@ -87,7 +90,41 @@ const refresh = action$ =>
             .mergeAll();
         });
 
+const handleBackgroundError = (action$, store) =>
+    action$.ofType(BACKGROUND_HAS_ERROR)
+    .filter(a => a.options.id === currentBackgroundLayerSelector(store.getState()).id)
+    .switchMap(() => {
```

Comment:
check the layer that produced error. If the error is caused by a supported layer, the cause must be not dependent by the maptype (e.g. API key not valid), so you should disable it at all in the background selector. Disable it at all for the future too.